packages/flutter/test/widgets/nested_scroll_view_test.dart

```diff
@@ -21,75 +21,78 @@ class _CustomPhysics extends ClampingScrollPhysics {
 }
 
 Widget buildTest({ ScrollController controller, String title:'TTTTTTTT' }) {
-  return new MediaQuery(
-    data: const MediaQueryData(),
-    child: new Directionality(
-      textDirection: TextDirection.ltr,
-      child: new Scaffold(
-        body: new DefaultTabController(
-          length: 4,
-          child: new NestedScrollView(
-            controller: controller,
-            headerSliverBuilder: (BuildContext context, bool innerBoxIsScrolled) {
-              return <Widget>[
-                new SliverAppBar(
-                  title: new Text(title),
-                  pinned: true,
-                  expandedHeight: 200.0,
-                  forceElevated: innerBoxIsScrolled,
-                  bottom: new TabBar(
-                    tabs: const <Tab>[
-                      const Tab(text: 'AA'),
-                      const Tab(text: 'BB'),
-                      const Tab(text: 'CC'),
-                      const Tab(text: 'DD'),
+  return new Directionality(
+    textDirection: TextDirection.ltr,
+    child: new MediaQuery(
+      data: const MediaQueryData(),
+      child: new Directionality(
+        textDirection: TextDirection.ltr,
```

Comment:
you ended up with two Directionalities here.

Reply:
Fixed.